src/pages/Index/Section/Personal/index.js

```diff
@@ -1,32 +1,59 @@
 import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
 import StartGuide from './StartGuide'
 import Cabinet from './Cabinet'
 import { Section, Container, Row } from '../index'
 import { Tab } from '../Trends'
 import Toggle from '../../../../components/VisibilityIndicator/Toggle'
 import styles from './index.module.scss'
 
+const tabByAnchor = () => {
+  const hash = window.location.hash
+
+  if (hash === TabType.CABINET.anchor) {
+    return TabType.CABINET.title
+  }
+}
+
 const LS_PERSONAL_TAB = 'LS_PERSONAL_TAB'
 const TabType = {
-  START_GUIDE: 'Quick Start Guide',
-  CABINET: 'Cabinet'
+  START_GUIDE: {
+    title: 'Quick Start Guide'
+  },
+  CABINET: {
+    title: 'Cabinet',
+    anchor: '#cabinet'
+  }
```

Comment:
Since `START_GUIDE` and `CABINET` are not equally typed objects, it's better to describe the `tab`-`hash` relationship in the different object, e.g.
```js
const TabHash = {
  [TabType.CABINET]: '#cabinet'
}
```

This will also remove a lot of unnecessary changes.